webpack/assets/javascripts/react_app/components/about/compute/index.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import Status from '../../common/status';
+import Table from '../../common/table';
+import helpers from '../../../common/helpers';
+
+class AboutComputeTable extends React.Component {
+  render() {
+    const columns = [
+      {
+        header: 'Name',
```

Comment:
string not extracted?

Reply:
extracted already in `<Table>` component